src/main/java/teammates/ui/webapi/request/FeedbackResponseCommentSaveRequest.java

```diff
@@ -0,0 +1,39 @@
+package teammates.ui.webapi.request;
+
+import javax.annotation.Nullable;
+
+/**
+ * The save request of a feedback response comment.
+ */
+public class FeedbackResponseCommentSaveRequest extends BasicRequest {
+
+    private String commentText;
+    @Nullable
+    private String showCommentTo;
```

Comment:
I don't like the current way of using string for `showCommentTo` and split it to construct feedbackPariticpantType. At least it should be something like `private List<FeedbackParticipantType> showCommentTo`. I think currently, we will leave it there and fix it when we migrate the comment feature in frontend.